Susi/Controllers/AboutViewController/AboutViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  AboutViewController.swift
+//  Susi
+//
+//  Created by Syed on 24/12/18.
+//  Copyright © 2018 FOSSAsia. All rights reserved.
+//
+
+import UIKit
+
+
+class AboutViewController: UIViewController, UITextViewDelegate {
+    
+     let susiUrl = "https://chat.susi.ai"
+     let contriUrl = "https://github.com/fossasia/susi_iOS/graphs/contributors"
+     let skillUrl = "www.skills.susi.ai"
+     let issueUrl = "https://github.com/fossasia/susi_iOS/issues"
+     let licenseUrl = "https://github.com/fossasia/susi_iOS/blob/master/LICENSE"
+
+    @IBOutlet weak var licenseTextField: UITextView!
+    @IBOutlet weak var introTextField: UITextView!
+    @IBOutlet weak var contributerTextField: UITextView!
+    @IBOutlet weak var skillTextField: UITextView!
+    @IBOutlet weak var issueTextField: UITextView!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        introTextField.delegate = self
+        contributerTextField.delegate = self
+        skillTextField.delegate = self
+        issueTextField.delegate = self
+        licenseTextField.delegate = self
+        introText()
+        contriText()
+        skillText()
+        issueText()
+        licenseText()
+        
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        let backButton: UIBarButtonItem = UIBarButtonItem(title: "Settings", style: .plain, target: self, action: #selector(back))
+        self.navigationItem.leftBarButtonItem = backButton
```

Comment:
Please add back narrow button too with `Settings`

Reply:
Since it is a barButton Item we can use either title or image for moving back , so i think it's better to use only back_arrow image
![screenshot 2018-12-28 at 1 42 30 am](https://user-images.githubusercontent.com/32494694/50493624-c0c99100-0a44-11e9-9b49-14b40bac21e4.png)